numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2891,9 +2893,11 @@ def add_newdoc(place, name, doc):
 
     See Also
     --------
-    floor_divide : Equivalent of Python ``//`` operator.
-    remainder : Equivalent of Python ``%`` operator.
-    modf : Equivalent to ``divmod(x, 1.0)``.
+    floor_divide : Equivalent to Python's ``//`` operator.
+    remainder    : Equivalent to Python's ``%`` operator.
+    modf         : Like ``divmod(x, 1.0)`` for positive ``x``, but returns
+                   ``(remainder, quotient)`` instead of
+                   ``(quotient, remainder)``.
```

Comment:
Not convinced the alignment of the colons buy anything here except extra spaces.